transaction/core/src/mint/constants.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! MobileCoin minting-related constants.
+
+/// Minimal nonce length.
+pub const NONCE_MIN_LENGTH: usize = 8;
```

Comment:
Why is the nonce variable-length?

Reply:
Because I have no idea how much data we're going to want to store in it - although I supposed it could be zero-padded. The idea is that it might get used to store some hash or something that is generated by the bridge.
Should I make it constant length? What would be a good length? 32? I suppose we could hash whatever comes from the bridge, unless we want it to be a bidirectional reference (e.g. the nonce pointing at the transaction on the bridge side that deposited funds for the minting).